pagopa/io-app
# feat: [#177590245] Added new component to shown assistance form in web view

## Short description
Created a new component to see if and how the assistance form works through a web view

**[Check this page](https://github.com/pagopa/io-app-assistance-form) to see what is and how to download/setup the assistance form**

## How to test
Try to use this new component in a correct scope to see if the assistance form works properly
